static/rage.js

```diff
@@ -162,6 +162,7 @@ function preselect_fields_based_on_params() {
   for (var i in checkboxes_on_by_default) {
     var cb_name = checkboxes_on_by_default[i];
     if (cb_name in params) continue;
+    set_auto_redraw();
```

Comment:
Is this the right place to call this? It's being called every time around the loop. Should it be called once, earlier?